tests/topotests/lib/pim.py

```diff
@@ -405,76 +405,6 @@ def find_rp_details(tgen, topo):
     return rp_details
 
 
-def configure_pim_force_expire(tgen, topo, input_dict, build=False):
```

Comment:
and now in the *third* version of this commit we remove a function that has absolutely nothing to do with the changes in the commit message.  This removal wasn't in the first two versions of this commit.  Stop playing games, I'm done with this whole thing.  You need to figure out how to properly break up your commits as well as properly test your code before you ask the community to review.  You are wasting our time.

Reply:
@donaldsharp , We discussed, I will update the PR on the basis of whatever clis are not used or needed in community specially for multicast. I did this change because we are updating pim.py file in this PR.

I did a thorough check and this API configure_pim_force_expire is no where used. I thought you will ok to remove all the APIs, which are using not needed CLIs in community. Please understand this is not to waste anyone's time. Just trying to enhance community's test capabilities. 

If you are not ok with the changes and tests added, we can close this PR and raise another PR post vacation. will keep note of your comments going forward. Thank you.